includes/class-ajax-data-retrieval.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * The core plugin class file
+ *
+ * Defines the functions necessary to register our AJAX functions
+ * in WordPress that will
+ *
+ * @since      0.0.2
+ *
+ * @package    WSUWP_Plugin_iDonate_Ajax_Data
+ * @author     Blair Lierman <blair.lierman@wsu.edu>
+ */
+class WSUWP_Plugin_iDonate_Ajax_Data {
+
+	/**
+	 * Initializes the actions necessary for hooking in our AJAX methods
+	 *
+	 * @since    0.0.2
+	 */
+	public function init() {
+
+		add_action( 'wp_ajax_nopriv_wsuwp_plugin_idonate_ajax_fund_search', array( $this, 'wsuwp_plugin_idonate_ajax_fund_search' ) );
+		add_action( 'wp_ajax_wsuwp_plugin_idonate_ajax_fund_search', array( $this, 'wsuwp_plugin_idonate_ajax_fund_search' ) );
```

Comment:
@jeremyfelt We are considering moving to the REST API (which I didn't realize was an option). Are there any other caveats to using it aside from making the post type public?